install/0000_90_cluster-version-operator_00_servicemonitor.yaml

```diff
@@ -16,3 +16,42 @@ spec:
   selector:
     matchLabels:
       k8s-app: cluster-version-operator
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    k8s-app: cluster-version-operator
+  name: cluster-version-operator
+  namespace: openshift-cluster-version
+spec:
+  groups:
+  - name: cluster-version
+    rules:
+    - alert: ClusterVersionOperatorDown
+      annotations:
+        message: ClusterVersionOperator has disappeared from Prometheus target discovery.
+      expr: |
+        absent(up{job="cluster-version-operator"} == 1)
+      for: 15m
+      labels:
+        severity: critical
+  - name: cluster-operators
+    rules:
+    - alert: ClusterOperatorDown
+      annotations:
+        message: ClusterOperator {{ $labels.name}} has not been available for 10 mins.
+      expr: |
+        cluster_operator_up{job="cluster-version-operator"}
```

Comment:
Do we actually report this?  From my reading of [this loop][1], it's just for ClusterOperator objects, and that's the operators reporting to the CVO, not the CVO itself (e.g. no `cluster-version-operator`) in [here][2].  Or is this "`cluster_operator_up` reported by the cluster-version-operator job" or some such?  Can you unpack this expression for me?

[1]: https://github.com/openshift/cluster-version-operator/blob/9a6c1125a72bcef276847e489eac8d580ccd1470/pkg/cvo/metrics.go#L241-L242
[2]: https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/pr-logs/pull/openshift_console/2261/pull-ci-openshift-console-master-e2e-aws/6623/artifacts/e2e-aws/must-gather/cluster-scoped-resources/config.openshift.io/clusteroperators/

Reply:
i'm not sure if other actions in the cluster can also emit something with name `cluster_operator_up`. so just want to make sure it's on the time series emitted by cluster-version-operator.

currently there's only one cvo, so namespace should be fine.

Reply:
> currently there's only one cvo, so namespace should be fine.

Couldn't end users create jobs with that name?  Can we rely on not scraping user namespaces?